### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by megalithic** [Pixabay/jQuery-flexImages#26]
Greetings @SimonSteinberger 

I've noticed there are a number of aging Issues and PRs for this project. Is it safe to say that even fixing bugs/submitting PRs to this project would be futile and that we should opt to just fork and create our own and republish away from this repo (given appropriate licensing of course)?

Thank you for your time and development of this project.

**Comment by novaeproject** [comment]
I'm currently using this project and am noticing some issues arising so I'd appreciate someone picking it up :D

**Comment by edinaldo12** [comment]
Yeah, still using this project. In the Firefox's beta latest version is returning an error though.

```
Firefox Developer Edition 117.0 Beta 8 (20230815180041)
Versão 117.0b8 (64 bits)
```

<img width="1437" alt="image" src="https://github.com/Pixabay/jQuery-flexImages/assets/82597670/298ce474-ac78-44a0-a115-937027657cdc">
